webapp/src/css/inbox.less

```diff
@@ -1231,6 +1231,13 @@ a.fa:hover {
   }
 }
 
+.partner-image {
+  img {
+    height: 150px;
+    width: 150px;
+  }
+}
+
```

Comment:
This will scale the images so if someone uploads something that is not square it will look awful!

Can you use `max-height` and `max-width` instead? This will mean the image is scaled down and the proportions are kept the same.

Also make sure there's some margin between images (~40px should be enough) as organisations are really sensitive about their logos.